Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/Features/Scanning/CameraPreviewRenderer.cs

```diff
@@ -91,11 +88,10 @@ public class CameraPreviewRenderer : ViewRenderer<CameraPreview, AutoFitTextureV
         // Orientation of the camera sensor
         private int mSensorOrientation;
 
-        // A {@link CameraCaptureSession.CaptureCallback} that handles events related to JPEG capture.
-        public CameraCaptureListener mCaptureCallback { get; set; }
-
         public Activity Activity { get; set; }
 
+        private static readonly SparseIntArray ORIENTATIONS = new SparseIntArray();
```

Comment:
Renaming "Orientations"

Reply:
Well, I see it's actually in some other places so no problem I guess... We should have added any rule to avoid these things

Reply:
done